os/arch/arm/src/imxrt/imxrt_clockconfig.c

```diff
@@ -92,6 +109,61 @@ const clock_usb_pll_config_t usb1PllConfig_BOARD_BootClockRUN = {
 	.src = 0,         /* Bypass clock source, 0 - OSC 24M, 1 - CLK1_P and CLK1_N */
 };
 
+const usb_phy_config_struct_t PhyConfig = {
```

Comment:
It is clock config, could you please explain why usb feature added here?

Reply:
No specific reason.
From boot sequence in iMXRT, usb phy initialization need to be done after completing clock configuration